src/AB1805_RK.cpp

```diff
@@ -583,15 +583,17 @@ bool AB1805::deepPowerDown(int seconds) {
     }
 
     // _log.trace("delay in case we didn't power down");   
-    unsigned long start = millis();
-    while(millis() - start < (unsigned long) (seconds * 1000)) {
-        _log.info("REG_SLEEP_CTRL=0x%2x", readRegister(REG_SLEEP_CTRL));
-        delay(1000);
+    if(loopToSleep) {
+        unsigned long start = millis();
+        while(millis() - start < (unsigned long) (seconds * 1000)) {
+            _log.info("REG_SLEEP_CTRL=0x%2x", readRegister(REG_SLEEP_CTRL));
+            delay(1000);
+        }
     }
 
     _log.error("didn't power down");
     delay(10);
-    System.reset();
+    // System.reset();
```

Comment:
why remove the reset if we cant power down?

Reply:
To deep reset on medela, you have to disconnect 5V and the battery in that order so no system reset. Its also just bad to have hidden resets like this